### ISSUE TITLE
WebM support

### ISSUE THREAD

**Issue by trashken** [sdelements/lets-chat#634]
Any quick way to add WebM support similar to jpg and gif?

**Comment by vwbusguy** [comment]
There's an upload filter: https://github.com/sdelements/lets-chat/wiki/Files

**Comment by vwbusguy** [comment]
If you mean to embed in the chat, you could update the imagePattern here: https://github.com/sdelements/lets-chat/blob/master/media/js/util/message.js , line 139.

**Comment by trashken** [comment]
Yes. Embed like when you paste an image link. Where do i add it in message.js?

**Comment by vwbusguy** [comment]
I mentioned above, line 139.  Specifically, in this part: (jpe?g|png|gif).  If you test it out and it works, you could submit it here as a pull request so others will have that functionality as well.
